nixos/modules/services/misc/zoneminder.nix

```diff
@@ -49,7 +49,7 @@ let
     # Database
     ZM_DB_TYPE=mysql
     ZM_DB_HOST=${cfg.database.host}
-    ZM_DB_NAME=${cfg.database.name}
+    ZM_DB_NAME=${if cfg.database.createLocally then user else cfg.database.username}
```

Comment:
Why? Why would you want a different user when having zoneminder create the database?